app/src/common/shared/com/igalia/wolvic/ui/widgets/menus/HamburgerMenuWidget.java

```diff
@@ -226,8 +231,7 @@ private void updateItems() {
                 .withIcon(R.drawable.ic_icon_resize)
                 .build());
 
-        // TODO this is hard to discover, consider adding a button in the URL bar instead
-        if (activeSession.getWebAppManifest() != null) {
+        if (!isKioskMode && activeSession.getWebAppManifest() != null) {
```

Comment:
instead of adding the condition to every if, maybe we can put them all together in a single `if (!kioskMode)` block

Reply:
Done.